xcodeproj/internal/xcodeproj_rule.bzl

```diff
@@ -796,6 +818,13 @@ labels must match transitive dependencies of the targets specified in the
 """,
             default = [],
         ),
+        "owned_extra_files": attr.label_keyed_string_dict(
+            allow_files = True,
+            doc = """\
+An optional dictionary of files to be added to the project. The key represents
+the file and the value is the label it should be associated with.
```

Comment:
```suggestion
the file and the value is the label of the target it should be associated with. These files won't be added to the project if the target is unfocused.
```

Reply:
This change seems to have been lost?

Reply:
Wrong merge conflicts, should be fixed now.